integration/spark/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -416,6 +416,31 @@ object CommonUtil {
     }
   }
 
+  /**
+   * This method will validate the cache expiration time specified by the user
+   *
+   * @param tableProperties table property specified by user
+   * @param propertyName property name
+   */
+  def validateCacheExpiration(tableProperties: Map[String, String], propertyName: String): Unit = {
+    var expirationTime: java.lang.Long = 0L
+    if (tableProperties.get(propertyName).isDefined) {
+      val value = tableProperties(propertyName)
+      val exceptionMsg = s"Invalid $propertyName value found: " +
```

Comment:
change this after changing unit to seconds

Reply:
OK.. will change in all impacted code.. once I change the timeunit

Reply:
changed